### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Redeyejedi35** [simplicialsoftware/nebulous-translations#659]
I found some errors in english on the simplicial software website, would this qualify me for translator tag? It should say "But it is not activated" And then the next two both should say "What do I do?"
![Screenshot_20240829_112550_Chrome](https://github.com/user-attachments/assets/b042a0be-ce47-45f6-b814-36861766c581)
![Screenshot_20240829_112535_Chrome](https://github.com/user-attachments/assets/7ef203ee-18bf-4117-94d5-caf1193a3a6b)
![Screenshot_20240828_045959_Chrome](https://github.com/user-attachments/assets/087ccb8d-294d-428a-b295-db7d763d0614)

**Comment by Jeii20** [comment]
@Redeyejedi35 
I don't think this would grant you the game translator tag, because the name itself says: "**Game** Translator". This tab doesn't belong to in-game, but to FAQ tab.

**Comment by Redeyejedi35** [comment]
But it's still about nebulous no?
